java/Solution_1323.java

```diff
@@ -0,0 +1,29 @@
+//
+
+public class Solution_1323
```

Comment:
I can merge this pr as soon as the name is changed to match the format and that the main function is taken out! :) 

I think the name should be `1323. Maximum 69 Number .java` judging by the solution